src/test/resources/superclass/input/com/example/Simple.java

```diff
@@ -0,0 +1,15 @@
+package com.example;
+
+class BigSimple {
+    public void printMessage() {
+        String x = "Hello";
```

Comment:
Why even include this line of code at all? The method is `void`, so its body can be empty.

(And, regarding our conversation the other day about `System.out.println`, the same thing applies to `java.lang.String`: it is technically a class in the JDK, and therefore a dependency if you include it here.)